src/modules/store.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable quotes */
+const InnerTodo = document.querySelector(".todo-inner");
+let isEdit = false;
+let editId = null;
+
+export default class display {
+  static getTodo = () => {
+    let Todo;
+    if (localStorage.getItem("todo") == null) {
+      Todo = [];
+    } else {
+      Todo = JSON.parse(localStorage.getItem("todo"));
+    }
+    return Todo;
+  };
+
```

Comment:
Well done implementing most of the requirements for this project. But there is something left. 

- [ ] As mentioned by the previous reviewer, based on this project's requirements _the index field value of a task should be set to the value of the new array length._ i.e  In a todo task object, the index field should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). 

Consider the image below, notice the `red` marking at the bottom right. I have added five tasks, and the first task should have an index value of 1 and not 0. The second task should have an index value of 2 and not 1, and so on. Kindly fix this. 
![image](https://user-images.githubusercontent.com/84156714/221061609-c6ba2a48-6755-4484-ab69-02201e55474d.png)